src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -551,6 +555,17 @@ static ConfigDef newConfigDef() {
             6,
             ConfigDef.Width.NONE,
             SNOWPIPE_STREAMING_MAX_CLIENT_LAG)
+        .define(
+            SNOWPIPE_STREAMING_MAX_CHANNEL_SIZE,
+            Type.LONG,
+            StreamingUtils.STREAMING_BUFFER_MAX_CHANNEL_SIZE_DEFAULT_BYTES,
+            ConfigDef.Range.atLeast(StreamingUtils.STREAMING_BUFFER_MAX_CHANNEL_SIZE_MINIMUM_BYTES),
+            Importance.LOW,
+            "Specify the maximum size (in bytes) of a single channel flush performed by the Ingest SDK",
+            CONNECTOR_CONFIG,
+            7, //TODO verify what this should be
```

Comment:
I'm not sure how important this is, but there is a 7 below already, see ENABLE_STREAMING_CLIENT_OPTIMIZATION_CONFIG, you can put it in the end and use a new number just in case

Reply:
Updated to 10, which was unused in the `CONNECTOR_CONFIG` group.